packaging/language/maven_artifact.py

```diff
@@ -81,6 +80,12 @@
         required: true
         default: present
         choices: [present,absent]
+    validate_certs:
+        description: If C(no), SSL certificates will not be validated. This should only be used on personally controlled sites using self-signed certificates.
```

Comment:
To be accurate - this can (and should) also be used with Maven Central. Can we update the documentation to reflect this information

http://blog.sonatype.com/2012/10/now-available-ssl-connectivity-to-central/#.Va-59RNVhBd